gammapy/image/maps.py

```diff
@@ -428,7 +428,7 @@ def solid_angle(self):
 
         dy = angular_separation(lon[:-1, :], lat[:-1, :],
                                 lon[1:, :], lat[1:, :])
-        return dx[1:, :] * dy[:, 1:]
```

Comment:
Replacing np.radians(coordinates.data.lon) with coordinates.data.lon.radian led to a test failure as np.radians(coordinates) returns Quantity(value, "rad") and coordinates.data.lon.radian np.nparray of floats.
So I had to specify that we return steradians